generator/src/main/java/dev/cdevents/generator/SchemaData.java

```diff
@@ -137,6 +139,22 @@ public void setSchemaURL(String schemaURL) {
         this.schemaURL = schemaURL;
     }
 
+    /**
+     *
+     * @return baseURI
+     */
+    public String getBaseURI() {
```

Comment:
We probably don't needs comments tbh. They don't really add any value.

Reply:
Needed, as we have current linter configured with `MissingJavadocMethod` for public methods.
https://checkstyle.org/checks/javadoc/missingjavadocmethod.html#MissingJavadocMethod

Reply:
I'd much rather add a check style suppression for this class, but that's a nit